apps/web/frontend/components/topNavigation/topNavigation.tsx

```diff
@@ -33,14 +33,12 @@ export const TopNavigation = (): JSX.Element => {
         {session.status !== 'authenticated' && (
           <TopNavigationLink onClick={() => signIn('github')}>Sign in</TopNavigationLink>
         )}
-
         <TopNavigationLink href="/time">Time</TopNavigationLink>
         <TopNavigationLink href="/sheet">Sheet</TopNavigationLink>
+        <div className="h-[20px]  w-[1px] bg-blue-400" />
```

Comment:
`h-[20px]`: arbitrary values should be avoided if possible. 

If you need the height to display anything at all, you can alternatively use CSS Flexbox to achieve that as well. The pipe is a child of a flexbox. You can tell a child to "stretch" the height.
More information here: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

If this makes the pipe too high, you can use vertical margin to adjust it again: e.g. `my-1`